File: programovani/ulohy/files/modelovani-obrazcu-i/ctverec-obrys.py
# Deklarace a inicializace
vel = 4 # Inicializace velikosti ctverce

# Vnejsi cyklus pro radky
for i in range(1, vel + 1):

    # Vnitrni cyklus pro jednotlive hvezdicky
    for j in range(1, vel + 1):           

        # Vypiseme hvezdicky
        if(i == 1 or i == vel):
            print("* ", sep="", end="")
        

        elif(j == 1 or j == vel):
            print("* ", sep="", end="")
       

        # Pro vnitrek vypiseme pouze mezery
        else:
            print("  ", sep="", end="")

    # Vypiseme novou radku    
    print(end="\n")

File: programovani/ulohy/files/modelovani-obrazcu-i/ctverec-obrys.cpp
#include <iostream>
using namespace std;


int main() {

    // Deklarace a inicializace
    const int vel = 4; // Inicializace velikosti ctverce

    // Vnejsi cyklus pro radky
    for(int i = 1; i <= vel; i++) {

        // Vnitrni cyklus pro jednotlive hvezdicky
        for(int j = 1; j <= vel; j++) {            

            // Vypiseme hvezdicky
            if(i == 1 || i == vel) {
                cout << "* ";
            }
            else if(j == 1 || j == vel) {
                cout << "* ";
            }

            // Pro vnitrek vypiseme pouze mezery
            else {
                cout << "  ";
            }

        }

        // Vypiseme novou radku
        cout << endl;

    }

}
